src/Scene/Scene.js

```diff
@@ -180,15 +160,63 @@ define('Scene/Scene', [
         this.gfxEngine.update();
     };
 
-    Scene.prototype.wait = function(timeWait) {
+    /**
+     * Notifies the scene it needs to be updated due to changes exterior to the
+     * scene itself (e.g. camera movement).
+     */
+    Scene.prototype.notifyChange = function() {
+        this.needsUpdate = true;
+        // request new frame in case the rendering loop was on hold
+        // force rendering to ensure interactivity
+        requestAnimationFrame(function(timestamp) { this.step(timestamp, true); }.bind(this));
+    };
 
-        var waitTime = timeWait ? timeWait: 20;
+    /**
+     * Subdivide layers and clean them if necessary
+     */
+    Scene.prototype.processLayers = function() {
```

Comment:
I'll try to find better names.
updateLayers updates the state of the layer for visualisation (tile visibility, sse...)
processLayers creates commands and cleans the quadtree.